src/test/resources/homework01/allTtest.xml

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+<suite name="Calculator all test">
+
+    <test name="Add test">
```

Comment:
Here you should not use declaration of classes to run, take a look for tag
See https://stackoverflow.com/questions/31851224/is-it-possible-to-run-several-testng-suite-files-in-one-configuration-in-intelli as example